Source/WebCore/bindings/js/JSWindowProxy.cpp

```diff
@@ -181,4 +184,164 @@ JSC::GCClient::IsoSubspace* JSWindowProxy::subspaceForImpl(JSC::VM& vm)
     return &static_cast<JSVMClientData*>(vm.clientData)->windowProxySpace();
 }
 
+struct FrameInfo {
```

Comment:
Since this is a temporary experiment that we'll want to get rid of ASAP, I think we should put all new code behind a build time flag to make the experiment-specific code obvious. It will make our life easier when we want to disable / get rid of the experiment.